lib/hls/hls_parser.js

```diff
@@ -2526,6 +2526,39 @@ shaka.hls.HlsParser = class {
     }
     return drmInfo;
   }
+
+  /**
+   * @param {!shaka.hls.Tag} drmTag
+   * @return {?shaka.extern.DrmInfo}
+   * @private
+   */
+  static playreadyDrmParser_(drmTag) {
```

Comment:
Could you include the link to the spec here?

Reply:
@TheModMaker Who manages the url shortener? I have seen that bit.ly is used but it is necessary to have an account ...